core/modules/taxonomy/views/views_handler_filter_term_node_tid_depth.inc

```diff
@@ -44,7 +44,7 @@ class views_handler_filter_term_node_tid_depth extends views_handler_filter_term
       return;
     }
     elseif (count($this->value) == 1) {
-      // Somethis $this->value is an array with a single element so convert it.
+      // This $this->value is an array with a single element so convert it.
```

Comment:
IMO, *Somethis* was supposed to be *Sometimes*. I would not use *This* when the next word is `$this`.

Reply:
Right. Good catch @kiamlaluno 👍🏼 ...why didn't I think of that? I'll need to check that and confirm based on the code that follows, but it sounds right.

Reply:
Fixed with https://github.com/backdrop/backdrop/pull/4575/commits/b8dd2af2921836a4057550fff3ef21ec44b118c8